### PR TITLE
[TESTNET-23] Aave apy api

### PR DESCRIPTION
**Pull request by Akhilleshgoswami**

### CODE REVIEW

File: cruize_operations/serilaizer/cruize_operations_serilaizer.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ class RepayToAaveRequestSerializer(serializers.Serializer):
 
 
 class FirebaeRequestSerializer(serializers.Serializer):
```
Comment (Prithviraj8) [inline cruize_operations/serilaizer/cruize_operations_serilaizer.py]: spelling mistake

File: cruize_operations/urls.py [hunk 2]
```diff
@@ -9,8 +9,9 @@
     path("transaction/save", CruizeOperations.as_view({"post": "save_transactions"})),
     path(
         "transaction/fetch",
-        CruizeOperations.as_view({"post": "fetch_user_transactions"}),
+        CruizeOperations.as_view({"get": "fetch_user_transactions"}),
```
Comment (Prithviraj8) [inline cruize_operations/urls.py:12]: Is this API accepting request parameters or request body ?

Comment (Prithviraj8) [reply cruize_operations/urls.py:12]: ignore this.. I just checked its query_params.. All good.

File: cruize_operations/views.py [hunk 3]
```diff
@@ -97,11 +106,21 @@ def set_price_floor(self, request):
...
+
+            result["result"] = price_floor_manager.set_price_floor(
                 asset_data["asset_name"], asset_data["days"]
             )
             return Response(result, status.HTTP_200_OK)
         except Exception as e:
             result["error"] = e
             return Response(result, status.HTTP_500_INTERNAL_SERVER_ERROR)
+
+    def get_asset_apy(self, request):
```
Comment (Prithviraj8) [inline cruize_operations/views.py]: can remove the word "get"

Comment (Prithviraj8) [reply cruize_operations/views.py]: or make it fetch_asset_apy

File: services/avve_asset_apy/aave_apy.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+from gql.transport.requests import RequestsHTTPTransport
+
+from utilities import cruize_constants
+
+
+class AaveApy:
+    def get_token_apys(self):
+
+        query = gql(cruize_constants.AVVE_SUB_GRAPH_QUERY)
+        sample_transport = RequestsHTTPTransport(
```
Comment (Prithviraj8) [inline services/avve_asset_apy/aave_apy.py]: sample_transport -> transport

File: services/avve_asset_apy/aave_apy.py [hunk 5]
```diff
@@ -0,0 +1,34 @@
+from gql import gql, Client
+from gql.transport.requests import RequestsHTTPTransport
+
+from utilities import cruize_constants
+
+
+class AaveApy:
+    def get_token_apys(self):
```
Comment (Prithviraj8) [inline services/avve_asset_apy/aave_apy.py]: Hey it seems we have both "token" & "asset" floating around our entire codebase. Let's stick to just one of them.
Either "token" or "asset", not both.

File: services/avve_asset_apy/aave_apy.py [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+            verify=True,
+            retries=3,
+        )
+        client = Client(transport=sample_transport)
+
+        response = client.execute(query)
+        apy_graph_data = response["reserves"]
+        subset_of_apy_token = set(cruize_constants.AAVE_APY_TOKENS)
+        apy_data = [i for i in apy_graph_data if i["symbol"] in subset_of_apy_token]
+        tokens_apy = {}
```
Comment (Prithviraj8) [inline services/avve_asset_apy/aave_apy.py]: tokens_apy or asset_apys or assets_apy or token_apys ? 🤣🤣

File: services/avve_asset_apy/aave_apy.py (hunk 5 above)
Comment (Akhilleshgoswami) [reply services/avve_asset_apy/aave_apy.py]: are you saying about this one ??

File: services/avve_asset_apy/aave_apy.py (hunk 4 above)
Comment (Akhilleshgoswami) [reply services/avve_asset_apy/aave_apy.py]: i have changed this no ??

**Review by LilFatFrank — DISMISSED** [review]

**Review by LilFatFrank — APPROVED** [review]

**Pull request merged**